internal/v1/server.go

```diff
@@ -552,6 +552,13 @@ func (h *Handlers) ComposeImage(ctx echo.Context) error {
 		return err
 	}
 
+	// check here for installer types
+	if composeRequest.Customizations != nil {
+		if composeRequest.Customizations.Users != nil && !strings.Contains(string(imageType), "installer") {
```

Comment:
I've just realized that this can be just one `if` statement with 3 conditions joined by `&&`.